chromadb/test/property/test_add.py

```diff
@@ -14,6 +14,7 @@ def api(request):
 
 
 @given(collection=strategies.collections(), embeddings=strategies.embedding_set())
+@settings(deadline=None)
```

Comment:
They seem to complete in reasonable time on my local machine. We can tighten as necessary.